packages/eas-cli/src/submissions/utils/builds.ts

```diff
@@ -0,0 +1,80 @@
+import gql from 'graphql-tag';
+
+import { graphqlClient } from '../../api';
+import { SubmissionPlatform } from '../types';
+
+const graphqlPlatform: Record<SubmissionPlatform, string> = {
+  [SubmissionPlatform.Android]: 'ANDROID',
+  [SubmissionPlatform.iOS]: 'IOS',
+};
+
+export async function getBuildArtifactUrlByIdAsync(
+  platform: SubmissionPlatform,
+  buildId: string
+): Promise<string> {
```

Comment:
```suggestion
): Promise<Build> {
```
Maybe this function should return the build identified by an id? This would make the function reusable in other places.

Reply:
I'm going to move and refactor those functions when building new GraphQL foundation that we discussed.